cmd/debos/debos.go

```diff
@@ -29,21 +37,32 @@ func main() {
 		TemplateVars  map[string]string `short:"t" long:"template-var" description:"Template variables"`
 	}
 
+	// Allow to run all deffer calls prior to os.Exit()
+	defer func() {
+		switch p := recover(); p {
+		case code0, code1:
+			os.Exit(p.(int))
+		default:
+			panic(p)
+		}
```

Comment:
Should we add a match for `nil` so we don't end up calling `panic(nil)` when no `panic()` has otherwise been called?